client/cli/src/commands/runcmd.rs

```diff
@@ -562,13 +563,34 @@ fn interface_str(
 	}
 }
 
-/// Default to verbosity level 0, if none is provided.
+/// Utility error for returning custom errors that implement the std::error::Error trait.
+#[derive(Debug)]
+struct TelemetryParsingError {
```

Comment:
This should be a strongly-typed enum. Errors that contain strings are generally not a good coding practice.
```suggestion
enum TelemetryParsingError {
    VerbosityLevelMissing,
}
```